decidim-core/app/views/decidim/reported_mailer/hide.html.erb

```diff
@@ -4,6 +4,22 @@
   <%= t(".report_html", url: reported_content_url ) %>
 </p>
 
+<p><b><%= t(".participatory_space") %></b></p>
+<p><%= link_to translated_attribute(@participatory_space.title), resource_locator(@participatory_space).url %></p>
+
+<p><b><%= t(".reason") %></b></p>
+<p><%= t(@report.reason, organization_name: organization_name(@participatory_space.organization), scope: "decidim.shared.flag_modal") %></p>
+
+<% if @report.details.present? %>
+  <p><b><%= t(".details") %></b></p>
+  <blockquote>
+    <%= @report.details %>
+  </blockquote>
+<% end %>
+
+<p><b><%= t(".content") %></b> (<%= t(".id") %>: <%= @reportable.id %>)</p>
```

Comment:
The ID just bring noise in this case

```suggestion
<p><b><%= t(".content") %></b></p>
```

Reply:
I understand this was added because is a copy paste from `decidim-core/app/views/decidim/reported_mailer/report.html.erb`
To be honest I didn't noticed until now. I'll remove it from there too. @carolromero can you confirm that we can drop this? I'm referring to the (`ID: 3571`) in this screenshot: 

![Screenshot of the mail](https://github.com/user-attachments/assets/e413f1b6-e2d2-45ea-bc08-9192df9ce211)